<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeightMapEditor</class>
 <widget class="QWidget" name="HeightMapEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Heightmap Graph Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="1">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="1">
        <widget class="QPushButton" name="buttonRender">
         <property name="text">
          <string>&amp;Render</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="buttonReset">
         <property name="text">
          <string>&amp;Reset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QGraphicsView" name="cellSelectView">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>560</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="0" column="2" rowspan="2">
      <widget class="QGraphicsView" name="renderView">
       <property name="minimumSize">
        <size>
         <width>320</width>
         <height>560</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="4">
         <widget class="QPushButton" name="buttonSave">
          <property name="text">
           <string>&amp;Save Graph</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="buttonCancel">
          <property name="text">
           <string>&amp;Cancel</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="buttonLoad">
          <property name="text">
           <string>&amp;Load Heightmap</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QTreeView" name="treeView">
        <property name="minimumSize">
         <size>
          <width>255</width>
          <height>0</height>
         </size>
        </property>
        <property name="headerHidden">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QtTreePropertyBrowser" name="propertyBrowser" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtTreePropertyBrowser</class>
   <extends>QWidget</extends>
   <header location="global">qttreepropertybrowser.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../Desktop/dev/ADWIF/editor/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>HeightMapEditor</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>389</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
